### PR TITLE
Newsweek home page clone

### PR DESCRIPTION
**Pull request by Blaise-Shyaka**
In this project, we have done the following:

- We set up linters
- We imported bootstrap CDN
- We wrote the markup for the page
- We styled the page using bootstrap classes
- We used a different CSS file to customize the bootstrap classes

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,2059 @@
...
+          <section class="d-md-none gap">
+            <div class="debate py-4">
+              <div class="h2-div font-weight-bold">THE DEBATE</div>
+              <div class="line"></div>
+              <div class="media the-debate">
+                <img
+                  class="d-flex mr-3"
+                  alt="Major"
+                  src="https://d.newsweek.com/en/full/1621698/major-neill-franklin-ret.png?w=150&h=150&f=fedb525be44c92fd2598c72f563e8f95"
+                  style="width: 64px; height: 64px;"
```
Comment (kevincappuccino) [inline index.html:356]: - [ ] Please remove the inline styling [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

File: index.html [hunk 2]
```diff
@@ -0,0 +1,2059 @@
...
+                    <span class="span-debate font-weight-bold"
+                      >KEVIN SABET</span
+                    >
+                  </p>
+                </div>
+                <img
+                  class="d-flex ml-3"
+                  src="https://d.newsweek.com/en/full/111059/kevin-sabet.png?w=150&h=150&f=2d885279d159e97deb11590f95b67c10"
+                  alt="Kevin Sabet"
+                  style="width: 64px; height: 64px;"
```
Comment (kevincappuccino) [inline index.html:390]: - [ ] Please remove the inline styling [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

File: index.html [hunk 3]
```diff
@@ -0,0 +1,2059 @@
...
+              <div class="line"></div>
+            </div>
+            <div class="opinion gap">
+              <div class="h2-div font-weight-bold py-4">OPINION</div>
+              <div class="media">
+                <img
+                  class="d-flex mr-3"
+                  alt="Slam"
+                  src="https://d.newsweek.com/en/full/1626435/salam-al-marayati.png?w=63&h=63&f=6b7145fbfece475556bde636cd82c729"
+                  style="width: 64px; height: 64px;"
```
Comment (kevincappuccino) [inline index.html:402]: - [ ] Please remove the inline styling [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

File: index.html [hunk 4]
```diff
@@ -0,0 +1,2059 @@
...
+                  <p class="font-weight-bold">BY SALAM AL-MARAYATI</p>
+                </div>
+              </div>
+              <hr />
+              <div class="media">
+                <img
+                  class="d-flex mr-3"
+                  alt="Dyllón"
+                  src="https://d.newsweek.com/en/full/1626108/dyllon-burnside.png?w=63&h=63&f=5cf9e387cc40a173b33ead8bbaf5558b"
+                  style="width: 64px; height: 64px;"
```
Comment (kevincappuccino) [inline index.html:417]: - [ ] Please remove the inline styling [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

File: index.html [hunk 5]
```diff
@@ -0,0 +1,2059 @@
...
+                  <p class="font-weight-bold">BY DYLLÓN BURNSIDE</p>
+                </div>
+              </div>
+              <hr />
+              <div class="media">
+                <img
+                  class="d-flex mr-3"
+                  alt="Mohammed"
+                  src="https://d.newsweek.com/en/full/1602983/mohammed-alsherebi.png?w=63&h=63&f=f3cb5702ecaac97c55ce6e0f1174eecb"
+                  style="width: 64px; height: 64px;"
```
Comment (kevincappuccino) [inline index.html:433]: - [ ] Please remove the inline styling [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

File: index.html [hunk 6]
```diff
@@ -0,0 +1,2059 @@
...
+                  <p class="font-weight-bold">BY MOHAMMED ALSHEREBI</p>
+                </div>
+              </div>
+              <hr />
+              <div class="media">
+                <img
+                  class="d-flex mr-3"
+                  alt="E. Wirth"
+                  src="https://d.newsweek.com/en/full/1604725/tom-rogers-timothy-e-wirth.png?w=63&h=63&f=c9021798f8a0bdb4f9c6625cd060f5ab"
+                  style="width: 64px; height: 64px;"
```
Comment (kevincappuccino) [inline index.html:448]: - [ ] Please remove the inline styling [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

File: index.html [hunk 7]
```diff
@@ -0,0 +1,2059 @@
...
+                  <p class="font-weight-bold">BY ILYA SHAPIRO</p>
+                </div>
+              </div>
+              <hr />
+              <div class="media">
+                <img
+                  class="d-flex mr-3"
+                  alt="Michael"
+                  src="https://d.newsweek.com/en/full/1625615/michael-makovsky.png?w=63&h=63&f=d1862ddf967d967845cb2ffb75032dc3"
+                  style="width: 64px; height: 64px;"
```
Comment (kevincappuccino) [inline index.html:479]: - [ ] Please remove the inline styling [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

File: index.html [hunk 8]
```diff
@@ -0,0 +1,2059 @@
...
+                  <p class="font-weight-bold">BY MICHAEL MAKOVSKY</p>
+                </div>
+              </div>
+              <hr />
+              <div class="media">
+                <img
+                  class="d-flex mr-3"
+                  alt="Cliff"
+                  src="https://d.newsweek.com/en/full/1625683/cliff-smith.png?w=63&h=63&f=5cce91c24e19cfadd1e4cd7eb5a6f0ae"
+                  style="width: 64px; height: 64px;"
```
Comment (kevincappuccino) [inline index.html:494]: - [ ] Please remove the inline styling [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

File: index.html [hunk 9]
```diff
@@ -0,0 +1,2059 @@
...
+          <section class="d-md-none gap">
+            <div class="more-stories mt-5">
+              <div class="h2-div font-weight-bold">MORE STORIES</div>
+              <hr />
+              <div class="media-1">
+                <img
+                  class="float-left mr-3"
+                  alt="64x64"
+                  src="https://d.newsweek.com/en/full/1626297/joe-biden-jill-dnc-2020.jpg?w=95&h=95&f=44299f0d2be506ecefd4079ee9206bdf"
+                  style="width: 95px; height: 95px;"
```
Comment (kevincappuccino) [inline index.html:613]: - [ ] Please remove the inline styling [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

File: index.html [hunk 10]
```diff
@@ -0,0 +1,2059 @@
...
+                    DNC celebrations with a speech calling for national unity.
+                  </p>
+                </div>
+              </div>
+              <div class="media-1">
+                <img
+                  class="float-left mr-3"
+                  alt="64x64"
+                  src="https://d.newsweek.com/en/full/1626322/goodyear.jpg?w=95&h=95&f=46d89aea47cb989d16a3f626b6a66d4c"
+                  style="width: 95px; height: 95px;"
```
Comment (kevincappuccino) [inline index.html:631]: - [ ] Please remove the inline styling [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

File: index.html [hunk 11]
```diff
@@ -0,0 +1,2059 @@
...
+                    enforcement.
+                  </p>
+                </div>
+              </div>
+              <div class="media-1">
+                <img
+                  class="float-left mr-3"
+                  alt="64x64"
+                  src="https://d.newsweek.com/en/full/1626230/bidens-granddaughters.jpg?w=95&h=95&f=dcaa9568f1a7cecc93620cb3c6fd0d16"
+                  style="width: 95px; height: 95px;"
```
Comment (kevincappuccino) [inline index.html:650]: - [ ] Please remove the inline styling [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

File: index.html [hunk 12]
```diff
@@ -0,0 +1,2059 @@
...
+                    in a video montage lending their support to his campaign.
+                  </p>
+                </div>
+              </div>
+              <div class="media-1">
+                <img
+                  class="float-left mr-3"
+                  alt="64x64"
+                  src="https://d.newsweek.com/en/full/1626414/moria.jpg?w=95&h=95&f=e7d961d612811f95904226c699333efa"
+                  style="width: 95px; height: 95px;"
```
Comment (kevincappuccino) [inline index.html:669]: - [ ] Please remove the inline styling [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

**Review by kevcap — CHANGES_REQUESTED** [review]
# [STATUS] CHANGES REQUIRED♻️

#### Woah, that's a good project 🤩! (One of the best I've ever seen and I mean it!) I liked it a lot, it matches mostly the requirements and breaks my heart to not approve it 😕. But I'm sure you both can easily improve that changes 💪 😎:


_Note_: That changes marked as [OPTIONAL] are not required to approve your project, but I strongly recommend that you guys go for that extra mile, in order to get more knowledge and have a better project to add at your portfolio.

## [REQUIRED CHANGES]
#### Good practices REQUIREMENTS (Those changes to make you code like a pro 👽)⤵
- [x] Files not related to the project should not be committed and added to .gitignore (e.g. .DS_Store, package-lock.json) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/gitignore.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/))
- [x] Kindly remember to remove all the unnecessarily repeated CSS code. Keeping your code DRY is an excellent practice that will help you a lot and make your code more professional. (Check this link form better understanding [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)
Take a look at this example:
image
Your HTML will be something like this: 
image
Apply the same logic above in **ALL** those repeated rules listed below:
image
- [x] Please remove all the inline styling [Good and Bad CSS Practices for Beginners](https://speckyboy.com/good-bad-css-practices/)) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/css_bad_practices_requirement.png) from Pathwright)

#### Clone REQUIREMENTS 👥 (Those adjusts to make your page really look like the real one)⤵
- [x] The desktop and Ipad version don't have some sections (U.S., World, Business, Tech & Science, Culture, Sports, so on); I know you guys did it because I can see it at the mobile version, there's an issue with the breakpoints, take care of that team! 😎;
- [x] Check and improve the size of the Subscribe and NEWSLETTER section;
image


## [OPTIONAL CHANGES]
- [x] Encourage you guys to use the `<main>` tag for the dominant content of the `<body>` of a document
- [x] Add the title of the video to the title of the page; (Newsweek - News, Analysis, Politics, Business, Technology)

## Summary
✨ Well done adding a favicon! 👏
✨ Your CSS looks very clean, organized, and professional! 🎨
✨ Overall your page looks almost identical congrats! 👌
✨ You guys have an eye to details and for sure got my attention, I'll keep my eyes on you! Keep up the good work! 🥇
🔧 Just have to implement that minor changes listed above, and I know that It will be easy for you both! 

### Please make the required changes and resubmit for a new code review once you are ready💪


## As always, I wish you @Blaise-Shyaka and @BrianSammit the very best and see you both at the top🚀

👨🏻‍💻 **TSE: Kevin Alves**

  

📫 Have any question about this review? Better call Kevin:

<a  href="mailto:kevinalves.js@gmail.com"> gmail</a> [Twitter Follow](https://twitter.com/intent/follow?screen_name=kevinnnmn) [](https://microverse-students.slack.com/team/UT86D3AP4)

**Review by MuriloRoque — APPROVED** [review]
# Status: Approved :green_circle: 

## Hello @Blaise-Shyaka and @BrianSammit :wave: 

## :clap: :clap: :clap: 

## I have no words, you guys are amazing, congratulations!

## You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**